front-end/src/components/App.js

```diff
@@ -25,7 +23,6 @@ class App extends Component {
                     <Route path="/settings" component={SettingsView} />
                     <Route exact path="/" render={() => <Redirect to="/newsfeed" />} />
                 </Switch>
-                <CookieView cookies={this.props.cookies} />
```

Comment:
if you are removing cookies, you might also want to remove withCookies wrapper from the last line, just export default AppContainer.

Reply:
Thank you. I have updated.